src/validation.cpp

```diff
@@ -1294,15 +1295,23 @@ void CChainState::InitCoinsCache(size_t cache_size_bytes)
 // `const` so that `CValidationInterface` clients (which are given a `const CChainState*`)
 // can call it.
 //
+// The file .ibd_complete is used to latch the datadir to IBD false across restarts
```

Comment:
If this moves forward, noting that a file description should be added to `doc/files.md` (alternatively, perhaps persist the latch in `settings.json`).

Reply:
I did this as a separate file so that users can easily add the file to override, with GetDataDir() it should be chain specific, no?

Reply:
Yes, I think they should both be.

Reply:
Users can easily override settings.json as well - they're just interpreted as command-line options.

So something like `--ibd_complete=0/1` would work